packages/gatsby-image/src/index.tsx

```diff
@@ -262,79 +429,106 @@ const noscriptImg = props => {
 // Earlier versions of gatsby-image during the 2.x cycle did not wrap
 // the `Img` component in a `picture` element. This maintains compatibility
 // until a breaking change can be introduced in the next major release
-const Placeholder = React.forwardRef((props, ref) => {
-  const { src, imageVariants, generateSources, spreadProps, ariaHidden } = props
-
-  const baseImage = (
-    <Img ref={ref} src={src} {...spreadProps} ariaHidden={ariaHidden} />
-  )
+const Placeholder = React.forwardRef(
+  (props: IPlaceholderProps, ref: React.Ref<HTMLImageElement>) => {
+    const {
+      src,
+      imageVariants,
+      generateSources,
+      spreadProps,
+      ariaHidden,
+    } = props
+
+    const baseImage = (
+      <Img ref={ref} src={src} {...spreadProps} ariaHidden={ariaHidden} />
+    )
 
-  return imageVariants.length > 1 ? (
-    <picture>
-      {generateSources(imageVariants)}
-      {baseImage}
-    </picture>
-  ) : (
-    baseImage
-  )
-})
-
-const Img = React.forwardRef((props, ref) => {
-  const {
-    sizes,
-    srcSet,
-    src,
-    style,
-    onLoad,
-    onError,
-    loading,
-    draggable,
-    // `ariaHidden`props is used to exclude placeholders from the accessibility tree.
-    ariaHidden,
-    ...otherProps
-  } = props
-
-  return (
-    <img
-      aria-hidden={ariaHidden}
-      sizes={sizes}
-      srcSet={srcSet}
-      src={src}
-      {...otherProps}
-      onLoad={onLoad}
-      onError={onError}
-      ref={ref}
-      loading={loading}
-      draggable={draggable}
-      style={{
-        position: `absolute`,
-        top: 0,
-        left: 0,
-        width: `100%`,
-        height: `100%`,
-        objectFit: `cover`,
-        objectPosition: `center`,
-        ...style,
-      }}
-    />
-  )
-})
+    return imageVariants.length > 1 ? (
+      <picture>
+        {generateSources(imageVariants)}
+        {baseImage}
+      </picture>
+    ) : (
+      baseImage
+    )
+  }
+)
 
-Img.propTypes = {
-  style: PropTypes.object,
-  onError: PropTypes.func,
-  onLoad: PropTypes.func,
-}
+const Img = React.forwardRef(
+  (props: IImgPropTypes, ref: React.Ref<HTMLImageElement>) => {
+    const {
+      alt,
+      sizes,
+      srcSet,
+      src,
+      style,
+      onLoad,
+      onError,
+      loading,
+      draggable,
+      // `ariaHidden`props is used to exclude placeholders from the accessibility tree.
+      ariaHidden,
+      ...otherProps
+    } = props
+
+    return (
+      <img
+        alt={alt}
```

Comment:
This seems like a change that's not part of the TS migration. I realize that sounds trivial but I'm not an expert on a11y issues and this is definitely in that ballpark and we should be careful about that. I kind of doubt this was a TS change.

I'm not against this change but could you double check and maybe add a separate issue / PR for it so an a11y expert can double check this? Thanks

Reply:
This is my mistake when migrating a test code.
This fix was not needed. Thank you very much.